website/templates/project/modal_add_component.mako

```diff
@@ -9,7 +9,8 @@
                 </div><!-- end modal-header -->
                 <div class="modal-body">
                     <div class="form-group">
-                        <input id="title" placeholder="Component Title" name="title" type="text" class='form-control'>
+                        <input id="title" maxlength="200" placeholder="Component Title" name="title"  type="text" class='form-control'>
+                        <div id="alert"></div>
```

Comment:
Do we have to use the ID "alert" here? It seems like that's a value that would be very likely to be used elsewhere.

I propose making it a class instead (`modal-alert`), and using a compound selector in `project.js` to grab it (`#newComponent .modal-alert`).